reviewrot.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import yaml
+import logging
+import logging.config
+from reviewrot import get_git_service
+from os.path import expanduser
+
+def main(fltr, value, duration):
+    """
+    starts execution
+    """
+    #TODO: Implement StreamHandler
+    # logging.config.fileConfig('log.conf')
+    log = logging.getLogger(__name__)
+    home = expanduser('~')
+    # read input from home directory for pull requests
+    with open(home + '/.config.yaml', 'r') as f:
+            config = yaml.load(f)
+    
+    #print config
+    for item in config['git']:
+        git = item.keys()[0].lower()
+        # get git service
+        git_service = get_git_service(git)
+        if item.values()[0] is not None:
```

Comment:
I think they could, for example, just because a user doesn't want to test something regarding gitlab for now doesn't mean they should get rid of the title 'gitlab'(especially now since we are planning to change the format of config).